services/interactions/src/models/interaction.ts

```diff
@@ -1,7 +1,7 @@
 import { Schema, model } from "mongoose";
 
 enum InteractionType {
-  EVENT = "event",
+  HEXATHON = "hexathon",
```

Comment:
So for these interactions here, we would actually keep them as event, as these interactions denote an interaction with an event not a hexathon (ie when you grab food, attend a mini event). so for this stuff, could u change it back?